sdks/universal-router-sdk/src/utils/constants.ts

```diff
@@ -294,6 +294,19 @@ export const CHAIN_CONFIGS: { [key: number]: ChainConfig } = {
       },
     },
   },
+  [60808]: {
+    weth: '0x4200000000000000000000000000000000000006',
+    routerConfigs: {
+      [UniversalRouterVersion.V1_2]: {
+        address: 'NA',
```

Comment:
These addresses were also missing from the spreadsheet

Reply:
Universal router is this address: 0x346239972d1fa486FC4a521031BC81bFB7D6e8a4 
https://explorer.gobob.xyz/address/0x346239972d1fa486FC4a521031BC81bFB7D6e8a4 (for blocknumber)